src/client/components/vision/camera/shaders/line_projection.frag

```diff
@@ -144,7 +146,7 @@ float rectilinearTheta(float r, float f) {
  * @return the distance from the optical centre when the point is projected onto the screen
  */
 float rectilinearR(float theta, float f) {
-  return f * tan(theta);
+  return f * tan(clamp(theta, 0.0, M_PI_2));
```

Comment:
A comment like what you have in the PR description would be good to know why it's clamped